### PR TITLE
Build.cleanup

### PR DESCRIPTION
**Pull request by rotty3000**
This uses the new (in bnd 5.0.0) `bnd-process-tests` maven mojo to simplify integration test setup.

### CODE REVIEW

File: org.osgi.test.junit4/test.bndrun [hunk 1]
```diff
@@ -30,10 +30,4 @@
 	ch.qos.logback.core,\
 	org.apache.felix.logback,\
 	slf4j.api
--runrequires: bnd.identity;id='${project.artifactId}-test'
--runbundles: \
```
Comment (bjhargrave) [inline org.osgi.test.junit4/test.bndrun]: I am not sure I like removing the `-runbundles` from the bndrun file. Is that necessary or a good idea?

File: org.osgi.test.junit5/test.bndrun [hunk 2]
```diff
@@ -37,22 +37,6 @@
 	org.apache.felix.logback,\
 	slf4j.api
 -runrequires: \
-	bnd.identity;id='${project.artifactId}-test',\
+	bnd.identity;id='${project.artifactId}-tests',\
 	bnd.identity;id='${junit-jupiter-engine.bsn}',\
 	bnd.identity;id='${junit-platform-launcher.bsn}'
--runbundles: \
```
Comment (bjhargrave) [inline org.osgi.test.junit5/test.bndrun]: I am not sure I like removing the `-runbundles` from the bndrun file. Is that necessary or a good idea?

File: org.osgi.test.junit4/test.bndrun (hunk 1 above)
Comment (rotty3000) [reply org.osgi.test.junit4/test.bndrun]: I mean it's recomputed each time already. So all this does is force us to have to commit changes to source anytime we make outside changes that affect the resolution result. All I personally care about for this is that tests actually run and pass :) we'd see a failure if the resolution was ever miscalculated due to human error.

... but, I'm not strongly arguing about it... just merely annoying for this scenario :)

Comment (rotty3000) [reply org.osgi.test.junit4/test.bndrun]: re-added it...

**Pull request merged**